internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -95,7 +95,10 @@ func convertSidecars(s map[string]*manifest.SidecarConfig, exposedPorts map[stri
 	sort.Strings(keys)
 	for _, name := range keys {
 		config := s[name]
-		imageURI := rc.PushedImages[name].URI()
+		var imageURI string
+		if image, ok := rc.PushedImages[name]; ok {
+			imageURI = image.URI()
+		}
```

Comment:
Not feedback but I found this snippet of code to be really interesting in Go!

The new code is definitely a more idiomatic way to approach the problem because it doesn't use what feels like a side effect of the language, but they actually have the exact same output. I was curious so I wrote this in the [Go Playground](https://go.dev/play/p/93jxmpKxkw0) to test.

❤️ Great catch on this code though since it's not actually part of the bug.